qt/src/SourceManager.cc

```diff
@@ -44,6 +54,30 @@ QDataStream& operator>>(QDataStream& ds, Source*&) {
 	return ds;
 }
 
+Source::Source(const QString& _path, const QString& _displayname, bool _isTemp)
+				: path(_path), displayname(_displayname), isTemp(_isTemp) {
+	// Check whether input PDF file is encrypted
+	if(path.endsWith(".pdf", Qt::CaseInsensitive)) {
+		std::unique_ptr<Poppler::Document> document(Poppler::Document::load(path));
+		if(document && document->isLocked()) {
+			bool ok = false;
+			QString text;
+			while(true) {
+				text = QInputDialog::getText(MAIN, _("Protected PDF"),
+											 _("Enter password for the PDF file"), QLineEdit::Password,
+											 "", &ok);
```

Comment:
Enter password for the PDF file**:** ;)

Reply:
I am not sure:
![image](https://user-images.githubusercontent.com/7355383/32444820-f1f7df3e-c314-11e7-9784-cd2792894539.png)
Is ':' required here?

Reply:
Ok this is probably debatable... Personally for strings that are not just titles, tooltips, menu entries or button labels I always terminate the string with some sort of punctuation (see also i.e. in the various UI files), so for consistency yes I'd add a colon here.

Reply:
Ok :-)

Fixed.